docs/components/pagination.md

```diff
@@ -0,0 +1,77 @@
+# Pagination
+
+Pagination enables users to navigate a set of records that are distributed across multiple pages. It is comprised of the following: 
+
+| Element | Description | Required? | 
+| --- | --- | --- | 
+| Page Buttons | Allows direct navigation to specific pages within a set | Yes | 
+| Previous / Next Buttons | Allows user to move forward or backwards in the set, depending on the current page | Yes |
+| Ellipses | Indicates a truncated range of pages | No | 
```

Comment:
Are the elements sorted any particular way? Should we put required elements first, or from large to small? (Saying it doesn't matter is also a valid answer 😄)

Reply:
I was thinking required first, then either "importance" or a-z